math_hurdlerActivity.py

```diff
@@ -31,15 +31,16 @@ def __init__(self, handle):
         self.build_toolbar()
 
         # Build the Pygame canvas.
-        self._pygamecanvas = sugargame.canvas.PygameCanvas(self)
+        self._pygamecanvas = sugargame.canvas.PygameCanvas(self,main = self.game.run)
```

Comment:
Add modules argument here, see sugargame/test activity for how it is done.

Also add pygame.font and pygame.mixer to modules, and remove the font.init() and mixer.init() from math_hurdler.py.

Style does not meet pep8, please run flake8.

Reply:
(but please don't fix all the other flake8 errors in this pull request; only fix the ones in what you have added.)